concepts/central-resource-sync/README.md

```diff
@@ -0,0 +1,135 @@
+# Software Design Description - Synchronization of Module Resources
```

Comment:
To reviewer:
I take this SDD document template from [here](https://github.wdf.sap.corp/DevProcess-Architecture/SoftwareDesignDescription), the main purpose of this document is trying to propose a design idea to address the business request from this [issue](https://github.com/kyma-project/lifecycle-manager/issues/913) for discussion, also it's for my architect curriculum on job task. If the content is not fit for this community repo, please let me know.